openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -333,6 +333,11 @@ components:
         region:
           description: A region where an Azure resource deployed.
           type: string
+        tags:
+          description: Tags for this Azure resource.
+          type: object
+          additionalProperties:
+            type: string

```

Comment:
since the `ApiAzureLandingZoneDeployedResource` model class is generated from the OpenApi definition, changing this definition adds a `tags` member to that model.